### PR TITLE
III-4071 Accept multiple issuers

### PR DESCRIPTION
**Pull request by bertramakers**
### Changed

- Changed how `iss` claim is validated, so we can validate multiple issuers

---
Ticket: https://jira.uitdatabank.be/browse/III-4071

Todo: Update config in appconfig and udb3-vagrant

### CODE REVIEW

File: src/Jwt/Symfony/Authentication/JwtAuthenticationProvider.php [hunk 1]
```diff
@@ -51,9 +51,9 @@ public function authenticate(TokenInterface $token)
             );
         }
 
-        if (!$this->decoderService->validateData($jwt)) {
+        if (!$this->decoderService->validateTimeSensitiveClaims($jwt)) {
             throw new AuthenticationException(
-                'Token claims validation failed. This most likely means the token is expired.'
+                'Token expired (or not yet usable yet).'
```
Comment (LucWollants) [inline src/Jwt/Symfony/Authentication/JwtAuthenticationProvider.php:63]: Nit: I think the last "yet" is not needed and "or not yet usable" is ok.

Comment (bertramakers) [reply src/Jwt/Symfony/Authentication/JwtAuthenticationProvider.php:63]: Yeah good spot, it's a typo!

**Review by LucWollants — APPROVED** [review]
Nice! Also thanks for the extra information about how leeway is handled 👌

**Comment by bertramakers** [comment]
https://github.com/cultuurnet/appconfig/pull/75 and https://github.com/cultuurnet/udb3-vagrant/pull/68 should be merged before we merge this one

**Pull request merged**